File: JEE/2016/python_comp/codes/ee16b1013.m
clear;
close;
clc;

x=linspace (-3*sqrt(3),3*sqrt(3), 200) ;
y=(3.0*(1-((x.*x)./27.0))).^0.5 ;
z=-(3.0*(1-((x.*x)./27.0))).^0.5 ;

plot (x,y,'LineWidth',4,x,z,'LineWidth',4) ;
grid;
xlabel('x');
ylabel('y');

print -deps -color ee16b1013.eps 

File: programming/EE1083/jee/2016/python_comp/codes/ee16b1013.py
import numpy as np
import matplotlib.pyplot as plt

x = np.linspace(-3*np.sqrt(3),3*np.sqrt(3),1000)
y = np.sqrt(3*(1-(x**2)/27))
z = -np.sqrt(3*(1-(x**2)/27))
plt.plot(x,y)
plt.plot(x,z)
plt.grid()
plt.xlabel('$x$')
plt.ylabel('$y$')
plt.savefig('../figs/ee16b1013.eps')
plt.show()
